com.unity.cinemachine/Runtime/Behaviours/CinemachineConfiner2D.cs

```diff
@@ -370,27 +370,17 @@ private void CalculateDeltaTransformationMatrix()
             return originalPath != null;
         }
 
-        internal float BakeProgress()
-        {
-            if (m_shapeCache.m_confinerOven != null)
-                return m_shapeCache.m_confinerOven.m_BakeProgress;
-            return 0f;
-        }
-
-        internal bool ConfinerOvenTimedOut()
-        {
-            return m_shapeCache.m_confinerOven != null 
-                && m_shapeCache.m_confinerOven.State == ConfinerOven.BakingState.TIMEOUT;
-        }
-    #endif
+        internal float BakeProgress() => m_shapeCache.m_confinerOven != null ? m_shapeCache.m_confinerOven.bakeProgress : 0f;
+        internal bool ConfinerOvenTimedOut() => m_shapeCache.m_confinerOven is { State: ConfinerOven.BakingState.TIMEOUT };
```

Comment:
is a null check required?  I'm unfamiliar with this syntax

Reply:
This `m_shapeCache.m_confinerOven is { State: ConfinerOven.BakingState.TIMEOUT };` is equivalent to
`m_shapeCache.m_confinerOven != null && m_shapeCache.m_confinerOven.State == BakingState.TIMEOUT`

It is new to me too, if you prefer, we can revert it to the more verbose normal syntax.

Reply:
I find that it's a bit opaque.  Unless there's a specific advantage to using it, the more explicit code is better and more readable imo.